tests/migrations/2016_06_27_000001_create_mediable_test_tables.php

```diff
@@ -22,7 +22,7 @@ public function up()
             $table->softDeletes();
         });
 
-        Schema::table('media', function (Blueprint $table) {
+        Schema::connection($this->getConnectionName())->table('media', function (Blueprint $table) {
```

Comment:
is it really necessary to specify this here? Are you not able to override getConnection like in the other migration files?